roles/foreman-proxy/defaults/main.yaml

```diff
@@ -0,0 +1,2 @@
+---
+foreman_proxy_image: quay.io/evgeni/foreman-proxy-rpm:3.12
```

Comment:
Which repository hosts this Containerfile?

Reply:
https://github.com/evgeni/foreman-container/blob/devel/Containerfile.proxy